Makefile

```diff
@@ -154,16 +159,15 @@ endif
 .PHONY: clean
 clean:
 	@if [ -d deps/jerryscript ]; then \
-		make -C $(JERRY_BASE) -f targets/zephyr/Makefile.zephyr clean; \
-		make -C $(JERRY_BASE) -f targets/zephyr/Makefile clean; \
+		make -C $(JERRY_BASE) -f targets/zephyr/Makefile clean BOARD=$(BOARD); \
```

Comment:
Okay, yeah make pristine rm's the outdir; I added a ZJS pristine target to call that too.